wiki/Article_styling_criteria/Writing/th.md

```diff
@@ -0,0 +1,321 @@
+
+### การจัดรูปแบบเวลา
+
+การจัดรูปแบบเวลาต้องทำโดยใช้รูปแบบต่อไปนี้:
+
+```markdown
+ชั่วโมงเลข 2 ตัว:นาทีเลข 2 ตัว เขตเวลา
+```
+
+เวลาเขียนในรูปแบบ 24 ชั่วโมง และตัวเลขเดี่ยวใดๆ จะต้องนำหน้าด้วยศูนย์ เขตเวลาจะต้องเขียนทันทีหลังจากเวลานั้น และควรอยู่ใน UTC+0 สำหรับเหตุการณ์ทั่วโลก หรือในเขตเวลาของกิจกรรมสำหรับกิจกรรมที่มีขนาดเล็กกว่า; หน่วยเวลาให้ใช้ `UTC` แทน `GMT`
+
+ตัวอย่างที่ไม่ดี:
+
+```markdown
+3:30 PM UTC
+22:30 (UTC+7)
+11:30
+```
+
+ตัวอย่างที่ดี:
+
+```markdown
+15:30 UTC
+22:30 UTC+7
+11:30 UTC-4
+```
+
+`UTC` (ที่ไม่มีตัวตามหลัง) ถูกบอกเป็นนัยว่าเป็น `UTC+0` อาจใช้อย่างใดอย่างหนึ่ง แต่การใช้งานต้องสอดคล้องกัน UTC ที่มีตัวตามหลังเป็น 0 จะต้องใช้สัญลักษณ์บวก (`+`)
+
+### การจัดรูปแบบวันที่และเวลา
+
+เมื่อใช้ทั้งวันที่และเวลาร่วมกัน ต้องเขียนวันที่ก่อน ตามด้วยเวลา เวลาต้องอยู่ระหว่างวงเล็บ (`(` และ `)`)
+
+ตัวอย่างที่ไม่ดี:
+
+```markdown
+October 25, 2016 at 11:45 UTC
+October 25, 2016 11:45 UTC
+```
+
+ตัวอย่างที่ดี:
+
+```markdown
+October 25, 2016 (11:45 UTC)
+```
+
+## โหมดของเกม
+
+โหมดเกมจะต้องเขียนดังนี้:
+
+- `osu!`
+- `osu!taiko`
+- `osu!catch`
+- `osu!mania`
+
+การอ้างถึงชื่อโหมดเกมแบบเก่า (เช่น `Catch the Beat`, `Taiko` และ `Mania`) อาจทำได้หากพูดถึงชื่อเดิมของโหมดเกมดังกล่าว
+
+## osu!
+
+ชื่อของเกม osu! ต้องไม่เป็นตัวพิมพ์ใหญ่หรือตัวเอียง โอสุ! ตราสินค้าอย่างเป็นทางการจะต้องไม่ใช้ช่องว่างใด ๆ ตัวอย่างได้แก่
+
+- `osu!academy`
+- `osu!api`
+- `osu!catch`
+- `osu!direct`
+- `osu!keyboard`
+- `osu!mania`
+- `osu!store`
+- `osu!stream`
+- `osu!supporter`
+- `osu!tablet`
+- `osu!taiko`
+- `osu!talk`
+- `osu!tourney`
+
+ชื่อผู้ใช้ที่มี `osu!` ซึ่งเป็นส่วนหนึ่งของชื่อหัวข้อจะต้องเป็นตัวพิมพ์ใหญ่ ตัวอย่างได้แก่
+
+- `osu! Alumni`
+- `osu! Champion`
+
+สำหรับเงื่อนไขอื่นๆ ทั้งหมด osu! ต้องถือว่าเป็น[คำนามที่ใช้ขยายคำนามอื่น](https://en.wikipedia.org/wiki/Noun_adjunct) นี่หมายถึงการเพิ่มช่องว่างระหว่าง osu! และคำนามที่ปรับเปลี่ยน ตัวอย่างได้แก่
+
+- `osu! tournaments`
+- `osu! community`
+- `osu! chat`
+- `osu! client`
+- `osu! wiki`
+
+---
+
+`osu!` ไม่ควรจบประโยค แต่ถ้าหากมันเป็นคนจบประโยคให้ใช้จุด full stop (`.`) หรือเครื่องหมายคำถามเพื่อให้เป็น `osu!.` หรือ `osu!?`
+
+`osu!` ต้องไม่ตามด้วยเครื่องหมายอัศเจรีย์ (เช่น `osu!!`) ตัวประโยคต้องถูกนำไปเขียนใหม่หากเกิดเหตุแบบนี้ขึ้น
+
+## คำศัพท์
+
+คำเหล่านี้ต้องสะกดดังนี้ (การวรรคต้องตรงกัน):
+
+- `approach circle`
+- `โหมดของเกม` (หรือ `โหมด`)
+- `ม็อดของเกม` (หรือ `ม็อด`)
+- `gameplay`
+- `hit burst`
+- `hit circle`
+- `hitsound`
+- `in-game`
+- `รูปแบบการเล่น`
+- `slider tick`
+- `slider ball`
+- `slider path`
+- `Kudosu` (ให้ใช้เป็นเป็นคำนามเฉพาะ)
+
+---
+
+บางคำมีรูปแบบต่างๆ ต้องใช้การสะกดคำที่ต้องการและมีดังนี้:
+
+- `creator` หรือ `mapper` แทน `beatmapper`.
+- `mapped` แทน `beatmapped`.
+- `BN` หรือ `Beatmap Nominators` เมื่อพูดถึง *ผู้เสนอบีทแมป*.
+- `sign in` แทน `log in`, เว้นแต่ชื่อของปุ่มหรือลิงก์จะระบุเป็นอย่างอื่น
+- `sign out` แทน `log out`, เว้นแต่ชื่อของปุ่มหรือลิงก์จะระบุเป็นอย่างอื่น
+- `register` แทน `sign up`, เว้นแต่ชื่อของปุ่มหรือลิงก์จะระบุเป็นอย่างอื่น
```

Comment:
@TicClick Sorry for the tag, but should this part be localized to Thai also?

Reply:
(please don't apologize in vain)
which part? I think that English articles suffer the most from inconsistent terms (especially terms that are inconsistent with these on the website), and this is not required for Thai. use whatever is widely used out there in the community

Reply:
Alright, Thanks